decidim-core/app/cells/decidim/card_m_cell.rb

```diff
@@ -34,7 +34,7 @@ def title
     end
 
     def description
-      decidim_sanitize(translated_attribute(model.description))
+      decidim_sanitize(html_truncate(translated_attribute(model.description), length: 100))
```

Comment:
Is 100 an appropiate length? It feels like it should maybe have a bit more content in there.

Reply:
It's the one we're using in other cards. Let's ship this, and we can increase it later